### ISSUE TITLE
Pen trail costumes in runtime

### ISSUE THREAD

**Issue by introlinux** [jmoenig/Snap--Build-Your-Own-Blocks#832]
In this thread: https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/issues/771
Jens commented about a hide (and cool) feature to convert a pen trail into a costume. Is it possible to do that using a block for converting trails in runtime and animate it later by code?

**Comment by jmoenig** [comment]
yes, that's totally possible. Hang on, I'm putting together a little example project that you'll be able to reuse...
